utils/keycodes.js

```diff
@@ -1,10 +1,14 @@
 export const BACKSPACE = 8;
 export const TAB = 9;
 export const ENTER = 13;
+export const CTRL = 17;
```

Comment:
This is no longer needed?

Reply:
I was wondering if I keep it or not (for the future). Treeshaking should get rid of it, at least theoratically